src/components/ToolTip/ToolTip.jsx

```diff
@@ -214,20 +214,20 @@ const ToolTip = createClass({
 				onMouseOver={this.handleMouseOverTarget}
 				onMouseOut={this.handleMouseOutTarget}
 			>
-				<Target className={cx('&', className)}>
+				<Target className={className}>
 					{target}
 				</Target>
 				<FlyOut
 					style={{
 						...flyOutStyle,
 						maxWidth: flyOutMaxWidth || flyOutStyle.maxWidth || 200
 					}}
-					className={flyOutCx('&', { [`${className}-FlyOut`]: className }, `&-${direction}`, `&-${alignment}`, `&-${kind}`)}
+					className={flyOutCx('className', '&', `&-${direction}`, `&-${alignment}`, `&-${kind}`)}
 					onMouseOver={this.handleMouseOverFlyout}
 					onMouseOut={this.handleMouseOutFlyout}
 				>
-					{isCloseable ? <CrossIcon onClick={this.handleClose} className={flyOutCx('&-close')}/> : null}
-					<h2 className={flyOutCx('&-title')}>{title}</h2>
+					{isCloseable ? <CrossIcon onClick={this.handleClose} className={flyOutCx('&-Close')}/> : null}
```

Comment:
If you want to make this uppercased, it should be `&-CrossIcon` according to our convention.